lib/msal-common/test/client/SilentFlowClient.spec.ts

```diff
@@ -306,17 +300,17 @@ describe("SilentFlowClient unit tests", () => {
             const config =
                 await ClientTestUtils.createTestClientConfiguration();
             const client = new SilentFlowClient(config, stubPerformanceClient);
-            const authResult = await client.acquireToken({
+            const authResult = await client.acquireCachedToken({
                 scopes: [],
                 account: testAccount,
                 authority: TEST_CONFIG.validAuthority,
                 correlationId: TEST_CONFIG.CORRELATION_ID,
                 forceRefresh: false,
             });
 
-            expect(authResult.account).toEqual(testAccount);
-            expect(authResult.idToken).toEqual(testIdToken.secret);
-            expect(authResult.accessToken).toEqual(
+            expect(authResult[0].account).toEqual(testAccount);
+            expect(authResult[0].idToken).toEqual(testIdToken.secret);
+            expect(authResult[0].accessToken).toEqual(
```

Comment:
I think you could add a test here to ensure that only one token was returned, and then set a const to equal the one token instead of using `[0]` three different times.